pkg/language/language_test.go

```diff
@@ -306,10 +307,16 @@ func TestDetect_ChromaTopLanguagesRetrofit(t *testing.T) {
 				"path/to/zshrc",
 				"path/to/.zshrc",
 				"path/to/PKGBUILD",
-				"testdata/bash",
 			},
 			Expected: heartbeat.LanguageBash,
 		},
+		"bash_from_file_contents": {
```

Comment:
It's readable for humans.
```suggestion
		"bash from file contents": {
```